man/LinReg.Rd

```diff
@@ -0,0 +1,90 @@
+\name{LinReg}
+\alias{LinReg}
+\alias{LinRegLA}
+\title{Simple Linear Regression}
+\usage{
+LinReg(model, particles = 1000, plot = FALSE)
+
+LinRegLA(model, particles = 1000, temperatures = seq(0, 1, 0.05)^5)
+}
+\arguments{
+\item{model}{Choice of regression model (1 for density as the predictor
+and 2 for adjusted density as the predictor).}
+
+\item{particles}{An integer specifying the number of particles.}
+
+\item{plot}{A boolean variable to determine whether to plot the posterior estimates.}
+
+\item{temperatures}{In likelihood annealing SMC the targets are defined as \eqn{P(y|\theta)^{\gamma_t}P(\theta)}
+where \eqn{0=\gamma_0\le \ldots \le \gamma_T = 1} can be referred to
+as the temperatures, \eqn{P(y|\theta)} is the likelihood and \eqn{P(\theta)}
+is the prior.}
+}
+\value{
+The \code{LinReg} function returns a \code{list} containing the final particle
+approximation to the target (\eqn{\theta} and the corresponding weights) as well as the logarithm
+of the estimated model evidence.
+
+The \code{LinRegLA} function returns a \code{list} containing the population
+of particles and their associates log likelihoods, log priors and weights at each iteration. The
+effective sample size at each of the iterations and several different
+estimates of the logarithm of the model evidence are also returned.
+}
+\description{
+A simple example based on estimating the parameters of a linear regression model
+using
+
+* Data annealing sequential Monte Carlo (\code{LinReg}).
+
+* Likelihood annealing sequential Monte Carlo (\code{LinRegLA}).
+}
+\details{
+Williams (1959) considers two competing linear regression models
+for the maximum compression strength parallel to the grain for radiata
+pine. Both models are of the form
+
+\eqn{y_i = \alpha + \beta (x_i - \bar{x}) + \epsilon_i},
+
+where \eqn{\epsilon_i ~ N(0,\sigma^2)} and \eqn{i=1,\ldots,42}.
+Here \eqn{y} is the maximum compression strength and \eqn{x} is either the density
+(model 1) or the adjusted density (model 2).
```

Comment:
The difference between the two models is in the choice of predictor: model 1 uses density and model 2 uses the adjusted density (taking into account the resin content). I've tried to make this a bit clearer in the commit I just did but I'm still not very happy with how it's documented.

I know that a more standard way to write this would be to write two sets of equations with different parameters. This is what's done in most papers:
y_i = \alpha + \beta (x_i - \bar{x}) + \epsilon_i where \epsilon_i ~ N(0,\sigma^2)
and
y_i = \gamma + \lambda (z_i - \bar{z}) + \epsilon_i where \epsilon_i ~ N(0,\tau^2)

I was trying to avoid using different notation for the two models because everything else is explained using the first notation and I didn't want to add subscripts because that looks messy in the .Rd format (unless I'm missing something).

If you think what I have now is unclear then please let me know. If we are going to change this work later so that it is more widely applicable then the description of this example would probably be less of an issue.